src/main/native/org/apache/commons/crypto/OpenSslInfoNative.c

```diff
@@ -94,30 +76,12 @@ static int load_library(JNIEnv *env)
     THROW(env, "java/lang/UnsatisfiedLinkError", msg);
     return 0;
   }
-  get_methods(env, openssl);
+#ifdef UNIX
```

Comment:
I'm sure we'll want this on Windows too at some point, but right now we need to figure out the Windows build in the first place, so this is ok.